apps/web/src/server/app/lpa/lpa-session.service.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Reset the LPA state in the session.
+ *
+ * @param {object} session – The session containing an LPA state.
+ * @returns {void}
+ */
+export const destroy = (session) => {
+	delete session.lpa;
+};
+
```

Comment:
I didn't have chance to add the documenting comments yet, apologies - I wanted to get the functionality itself working before I finish on the project